tests/tools/hash_utils.cc

```diff
@@ -8,8 +8,10 @@
 #include <sstream>
 #include <iostream>
 #include <unordered_set>
+#include <array>
+#include <unordered_map>
 
-// // TODO: add asserts
+// TODO: add asserts

```

Comment:
on what? (also, if it takes twenty or less minutes we should just do it before merging)

Reply:
lol this is my old comment let’s delete this